debugging/helper.js

```diff
@@ -108,7 +108,8 @@ const checkLeader = () => {
   } else if (response.error) {
     fatal("Got error while checking for leader agency: " + response.errorMessage);
   }
-  return response;
+  const stores = httpWrapper('GET', '/_api/agency/stores');
+  return [response, stores];
```

Comment:
Hey @kvahed 
You do know that this change actually breaks all the following scripts:

```
debugging/tasks/clear-maintenance.js
debugging/tasks/create-move-plan.js
debugging/tasks/dump.js
debugging/tasks/show-move-shards.js
debugging/tasks/show-supervision.js
debugging/helper.js
```

only:
```
debugging/tasks/analyze.js
```

has been adapted.

=> All repair scripts are now broken